### PR TITLE
Implement repeating fields in edit forms

### PR DESCRIPTION
**Pull request by awead**
## Description

Enables adding and removing multiple repeating fields in our edit forms. Fields that are marked `multiple` in the data dictionary have this feature enabled.

Connected to #797 

## Changes

* install webpacker
* ECMA6 Javascript for cloning and removing nodes in forms
* Integration testing using webdriver with a headless Chrome browser

### CODE REVIEW

File: spec/cho/collection/archival_collections/edit_spec.rb [hunk 1]
```diff
@@ -11,8 +11,8 @@
     it 'updates an existing work with new metadata' do
       visit(edit_archival_collection_path(resource))
       expect(page).to have_field('Description', type: 'textarea', with: 'Sample archival collection')
-      fill_in('archival_collection[title]', with: 'Updated Archival Collection Title')
-      fill_in('archival_collection[description]', with: 'Updated archival collection description')
+      fill_in('archival_collection[title][]', with: 'Updated Archival Collection Title')
```
Comment (cam156) [inline spec/cho/collection/archival_collections/edit_spec.rb]: Shouldn't title and description be singular?

File: spec/cho/work/submissions/edit_spec.rb [hunk 2]
```diff
@@ -11,15 +11,15 @@
   context 'with all the required metadata' do
     it 'updates an existing work with new metadata' do
       visit(edit_work_path(resource))
-      expect(page).to have_field('work_submission_creator_role', with: resource.creator.first[:role])
-      expect(page).to have_field('work_submission_creator_agent', with: resource.creator.first[:agent])
+      expect(page).to have_field('work_submission_creator__role', with: resource.creator.first[:role])
```
Comment (cam156) [inline spec/cho/work/submissions/edit_spec.rb]: This seems like an odd change

File: spec/support/database_cleaner.rb [hunk 3]
```diff
@@ -2,27 +2,31 @@
...
+                                 :transaction
+                               end
   end
 
   config.before do
     DatabaseCleaner.start
   end
 
-  config.after do
+  # This is horrendous, but JS tests don't seem to work with the truncation strategy, so
```
Comment (cam156) [inline spec/support/database_cleaner.rb]: Should this be the deletion strategy? `but JS tests don't seem to work with the :deletion strategy`

File: spec/cho/collection/archival_collections/edit_spec.rb (hunk 1 above)
Comment (awead) [reply spec/cho/collection/archival_collections/edit_spec.rb]: @cam156 They're not in the data dictionary for test. However, in dev and prod title is singular while description is multiple. 🤷‍♂️

File: spec/cho/work/submissions/edit_spec.rb (hunk 2 above)
Comment (awead) [reply spec/cho/work/submissions/edit_spec.rb]: @cam156 Yeah, it looks odd. This is a side-effect from making creator multiple. The name is now `work_submission[creator][][role` which translates into an id of `work_submission_creator__role`.

File: spec/support/database_cleaner.rb (hunk 3 above)
Comment (awead) [reply spec/support/database_cleaner.rb]: @cam156 It used to be deletion, but truncation is the recommend way. https://github.com/DatabaseCleaner/database_cleaner#rspec-with-capybara-example

**Pull request merged**